tests/e2e/utils/data_fetcher/idempotency.py

```diff
@@ -0,0 +1,57 @@
+import boto3
+from retry import retry
+
+
+class DynamoDB:
```

Comment:
Can we call it DynamoDBFetcher?

```suggestion
class DynamoDBFetcher:
```

Reply:
I removed this class. As you said: this is very simple to use an intermediate class.